nft/nfts.yaml

```diff
@@ -30,9 +30,34 @@ paths:
         - $ref: nfts_body.yaml#/withMetadata
           description: 'True by default - If boolean is set to true the query will include metadata for each returned token '
         - $ref: nfts_body.yaml#/filters
-          description: |-
+          description: |- 
             Array of filters (as ENUMS) that will be applied to the query. NFTs that match one or more of these filters will be excluded from the response. Filter Options: 
              - SPAM: NFTs that have been classified as spam. Spam classification has a wide range of criteria that includes but is not limited to emitting fake events and copying other well-known NFTs.
+      x-readme: 
+        samples-languages:
+          - javascript
+          - curl
+          - python
+          - go
```

Comment:
Instead of defining these in every method we can define them in a components section at the bottom and use it across every method. We can't do this for the code-sample itself but it should work for the languages at least. Does that make sense?

Reply:
I wasn't able to get this working! Can you try it on your machine and send me the code if it works?